modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/SequenceSearch.scala

```diff
@@ -58,9 +59,8 @@ object SequenceSearchResultsBody {
               <.td(u.instrument),
               <.td(
                 ^.cls := "collapsing",
-                <.button(
-                  ^.cls := "circular ui icon button",
-                  Icon(Icon.Props("plus", onClick = onAdding(p.searchResults, u)))
+                Button(Button.Props(icon = true, circular = true, onClick = onAdding(p.searchResults, u)),
+                  Icon(Icon.Props("plus"))
```

Comment:
Is there a way to just specify the `Icon` and not have to also set the property `icon`?